src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -1041,6 +1041,26 @@ function bindCopyEvents() {
     });
 }
 
+function bindAddTemplateQnEvents() {
+    $('#addTemplateQuestionModal .panel-title').click(function (e) {
+        if (!$(e.target).is('input')) {
+            $(this).closest('.panel').find('.panel-collapse').collapse('toggle');
+        }
+    });
+
+    $('[id^="addTemplateQuestion-"]').click(() => {
+        const numCheckboxChecked = $('input[name="templatequestionnum"]:checked').length;
+        const $button = $('#button_add_template_submit');
+
+        $button.prop('disabled', numCheckboxChecked <= 0);
+        $button.click(() => {
```

Comment:
The button event should be bound just once?

Give `disabled` to the button already make the button not clickable.

Reply:
The `$button.click()` binding should be done inside bindAddTemplateQnEvents() rather than inside another click handler.